pilot/pkg/xds/debug.go

```diff
@@ -378,11 +378,13 @@ func (s *DiscoveryServer) endpointz(w http.ResponseWriter, req *http.Request) {
 	writeJSON(w, resp, req)
 }
 
+var DistributionTrackingDisabledMessage = "Pilot Version tracking is disabled.  Please set the " +
+	"PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING environment variable to true to enable."
+
```

Comment:
I suggest that change the message as below:
"Pilot Version tracking is disabled.  Please enable it by setting " +
	"PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING environment variable as true."